From 3c93cda8211efa01128d48950f0d6ee5233c5b9b Mon Sep 17 00:00:00 2001
From: stha09 <51720730+stha09@users.noreply.github.com>
Date: Thu, 6 May 2021 18:31:30 +0200
Subject: [PATCH] IWYU: include limits for std::numeric_limits (#253)

Builds with the latest layers fail as a result of stricter compiler
checks. This patch backports a fix for a missing include statement in
the third party dependency on ruy.

Upstream-Status: Backport [https://github.com/google/ruy/commit/3c93cda8211efa01128d48950f0d6ee5233c5b9b]

Signed-off-by: Tony Tascioglu <tony.tascioglu@windriver.com>
---
 third_party/ruy/src/ruy/block_map.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/third_party/ruy/src/ruy/block_map.cc b/third_party/ruy/src/ruy/block_map.cc
index 6c01e52..e04e7af 100644
--- a/third_party/ruy/src/ruy/block_map.cc
+++ b/third_party/ruy/src/ruy/block_map.cc
@@ -17,6 +17,7 @@ limitations under the License.
 
 #include <algorithm>
 #include <cstdint>
+#include <limits>
 
 #ifdef RUY_MAKEBLOCKMAP_DEBUG
 #include <cstdio>
-- 
2.31.1

